scripts/npm/prepublish/lib/docs.js

```diff
@@ -0,0 +1,46 @@
+var exec = require('child-process-promise').exec;
+var util = require('../../lib');
```

Comment:
This can just be `var util = require('./');` because all the util functions are in `index.js` which is in the same directory as this script. Buuuuuuuuut that won't work because after `docs: require('./docs')` is added to that file, it will have a circular dependency. So it's probably easiest to just replace this line with:

```
var util = {
  printLn: require('./print'),
  git: require('./git')
}
```

Those are the only two util functions needed in this file.